package.json

```diff
@@ -0,0 +1,5 @@
+{
+  "dependencies": {
+    "aiconfig": "^1.1.1"
+  }
```

Comment:
I think most of our cookbooks now are in python so can we remove? Can you share the documentation you followed so we can learn how you got here? Thanks!

Reply:
I was following the readme here to get started https://github.com/lastmile-ai/aiconfig